pom.xml

```diff
@@ -23,159 +23,24 @@
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
-    <groupId>org.apache</groupId>
-    <artifactId>apache</artifactId>
+    <groupId>cloud.cosmic</groupId>
+    <artifactId>cosmic</artifactId>
     <version>11</version>
+    <relativePath>../pom.xml</relativePath>
   </parent>
 
-  <groupId>org.apache.cloudstack</groupId>
-  <artifactId>cloudstack</artifactId>
-  <version>4.8.0-SNAPSHOT</version>
+  <groupId>cloud.cosmic</groupId>
+  <artifactId>cosmic-core</artifactId>
+  <version>5.0.0-SNAPSHOT</version>
   <packaging>pom</packaging>
   <name>Apache CloudStack</name>
   <description>Apache CloudStack is an IaaS (“Infrastructure as a Service”) cloud orchestration platform.</description>
   <url>http://www.cloudstack.org</url>
-  <scm>
-    <connection>scm:git:https://git-wip-us.apache.org/repos/asf/cloudstack.git</connection>
-    <developerConnection>scm:git:https://git-wip-us.apache.org/repos/asf/cloudstack.git</developerConnection>
-  </scm>
-  <issueManagement>
-    <system>jira</system>
-    <url>https://issues.apache.org/jira/browse/CLOUDSTACK</url>
-  </issueManagement>
 
   <prerequisites>
     <maven>3.0.4</maven>
   </prerequisites>
```

Comment:
let's add the issuemanagement and scm when we have something to define there.
willing to move to 3.3.9 but will need to test the build again